SPECS/kernel/kernel.spec

```diff
@@ -28,7 +28,7 @@
 Summary:        Linux Kernel
 Name:           kernel
 Version:        5.15.131.1
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
I think we might also want to bump the release in `kernel-headers` & `kernel-signed` to keep things in parity.

Reply:
yes...build revealed that issue.  Done.